google-cloud-datastore/README.md

```diff
@@ -212,6 +220,7 @@ Apache 2.0 - See [LICENSE] for more information.
 [LICENSE]: https://github.com/GoogleCloudPlatform/google-cloud-java/blob/master/LICENSE
```

Comment:
I don't quite get what you mean - are you suggesting that the license section be stripped down only to the link?